server/utils_test.go

```diff
@@ -41,3 +44,42 @@ func Test_parseSchedule(t *testing.T) {
 		})
 	}
 }
+
+func Test_nextWeekdayDateInWeekSkippingDay(t *testing.T) {
+	var patch, err = mpatch.PatchMethod(time.Now, func() time.Time {
+		return time.Date(2021, 11, 15, 00, 00, 00, 0, time.UTC)
+	})
```

Comment:
I was thinking we would pass in the value for `now` in the function we're testing this with, rather than implicitly patching the value here. @sanjaydemansol What do you think?

We would modify `nextWeekdayDateInWeekSkippingDay` to accept `now` as a parameter:

```go
now, _ := time.Date(2021, 11, 15, 00, 00, 00, 0, time.UTC)

got, err := nextWeekdayDateInWeekSkippingDay(now, tt.args.meetingDays, tt.args.nextWeek, tt.args.dayToSkip)
```